integreat_cms/cms/models/abstract_content_translation.py

```diff
@@ -431,7 +431,7 @@ def translation_state(self):
             # If the page does not have a major public version, it is considered "missing" (keep in mind that it might
             # have draft versions or public versions that are marked as "minor edit")
             return translation_status.MISSING
-        if translation.currently_in_translation:
+        if self.currently_in_translation:
```

Comment:
```suggestion
        # For "currently in translation", we consider the latest version instead of the latest major version
        if self.currently_in_translation:
```
In case we change something related to the translation state in the future (which isn't that unrealistic I think :grin:), I'd like to make sure we remember why we chose a different version here than for the other states.